apps/docs/troubleshooting/extglossary.md

```diff
@@ -0,0 +1,98 @@
+
+![PreMiD Functions](/guide-images/gu-p1-pmsetting.png)
+
+**Heartbeat**
+
+- Heartbeat is a diagnostic tool. In _future versions of PreMiD_ this will show some fancy information and statistics.
+
+**Developer Mode**
+
+- A diagnostic and development tool. Most users will never use this. Toggling this on will open the websocket port of PreMiD.
+
+**Sync Language**
+
+- This option allows for the user to mandate their language settings be set between all presences based on the choice made in the PreMiD application.
+- If this is not selected, the user may mandate different languages for separate presences.
+
+**Use Playing Status**
+
+- _[Soon to be depreciated.]_ This tells PreMiD to use "Playing" status over use "Listening" status or use another type of action.
+- The extension is currently in the process of being updated. When this happens, the function will no longer exist-- as it is able to be controlled by a presence rather than the overall application.
+
+**Prefer App**
+
+- _[Soon to be depreciated.]_ When PreMiD utilized an application as well as an extension bundle, this would notify the extension of the application being installed on the machine, sending information to the local machine (or LocalHost) to transmit data to the application. That information would then be pushed directly to Discord. Due to Discord development and policy changes, there is unfortunately, no longer an application.
+- The extension is currently in the process of being updated. When this happens, the function will no longer exist.
+
+## Bundled Presence Functions
+
+> [!NOTE]
+> This is not a fully comprehensive list, but should give a relatively comfortable idea of how presences function with different options.
+
+<sub>This includes the presences that are included with the application on install. Other presences available will have the same or similar options, with the same functionality.</sub>
+
+**Privacy Mode**
+
+- Included in almost all PreMiD presences.
+- All presences with this option will function the same.
+- Allows a user to hide what they are watching while still showing the specific presence application.
+  - _EG: Watching Netflix for 2:49_
+
+![YouTube Privacy](/guide-images/gu-p4-privorte.png)
+
+**Privacy Overwrite Button**
+
+- _YouTube Specific_
+- Allows a user to implement a small eye-shaped button on the media-player that can control privacy-mode without opening the PreMiD extension.
+- This can be toggled on-and-off.
+
+**Privacy Overwrite TTL**
+
+- _YouTube Specific_
+- When using the Privacy Overwrite option, it can be set to have a total lifespan.
+- Has no function if not using Privacy Mode.
+
+**Show Live streams**
+
+- _Twitch Specific_
+- Shows information in regards to live broadcasts when on the applicable page.
+
+**Show Movies/Series**
+
+- _Netflix Specific_
+- When enabled will show information in regards to Movies or Series while watching.
```

Comment:
Why mention activity specific settings?

Reply:
@Espressocodes

Reply:
Added due to FAQ-nature of troubleshooting guide. It was a collection of specifics that are on mostly default presences that cause issues with some less tech-minded installers.

Reply:
Questions about specific presence settings are frequently asked in the discord,
Being able to point users to the docs is a huge time saver

Reply:
feel free to purge it. It's whatever y'all want. I just write